libs/features/mas/web-components/src/hydrate.js

```diff
@@ -237,6 +251,7 @@ export async function hydrate(fragmentData, merchCard) {
     merchCard.removeAttribute('badge-color');
     merchCard.removeAttribute('badge-text');
     merchCard.removeAttribute('size');
+    merchCard.removeAttribute('daa-ll');
```

Comment:
why would there be a daa-ll attribute there?

Reply:
will fix to 'daa-lh'